docs/src/modules/components/DiamondSponsors.js

```diff
@@ -86,7 +86,7 @@ export default function DiamondSponsors() {
             (theme) => ({
               '& a': {
                 width: '100%',
-                height: { xs: 40, sm: 50 },
+                height: { xs: 40, sm: 45 },
```

Comment:
I'm struggling with the diamond sponsors at the bottom, the UX issue for me is when I expand a side nav I don't have enough vertical height to browse the list comfortably (desktop)

Reply:
Seems aligned with what a community member is feeling too (https://github.com/mui/material-ui/issues/38033)...  I'm considering changing it.